resources/kcp/charts/keda/README.md

```diff
@@ -0,0 +1,147 @@
+| `nodeSelector`                                             | Node selector for pod scheduling ([docs](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/)) | `{}` |
+| `tolerations`                                              | Tolerations for pod scheduling ([docs](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)) | `{}` |
+| `affinity`                                                 | Affinity for pod scheduling ([docs](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)) | `{}` |
+| `priorityClassName`                                        | Pod priority for KEDA Operator and Metrics Adapter ([docs](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)) | `` |
+| `env`                                                      | Additional environment variables that will be passed onto KEDA operator and metrics api service | `` |
+| `http.timeout` | The default HTTP timeout to use for all scalers that use raw HTTP clients (some scalers use SDKs to access target services. These have built-in HTTP clients, and the timeout does not necessarily apply to them) | `` |
+| `service.annotations`                                      | Annotations to add the KEDA Metric Server service | `{}`                                        |
+| `service.portHttp`                                         | Service HTTP port for KEDA Metric Server service | `80`                                        |
+| `service.portHttpTarget`                                   | Service HTTP port for KEDA Metric Server container | `8080`                                        |
+| `service.portHttps`                                        | HTTPS port for KEDA Metric Server service | `443`                                        |
+| `service.portHttpsTarget`                                  | HTTPS port for KEDA Metric Server container | `6443`                                        |
+| `prometheus.metricServer.enabled`                          | Enable metric server prometheus metrics expose | `false`
+| `prometheus.metricServer.port`                             | HTTP port used for exposing metrics server prometheus metrics | `9022`
+| `prometheus.metricServer.portName`                         | HTTP port name for exposing metrics server prometheus metrics | `metrics`
+| `prometheus.metricServer.path`                             | Path used for exposing metric server prometheus metrics | `/metrics`
+| `prometheus.metricServer.podMonitor.enabled`               | Enable monitoring for metric server using podMonitor crd (prometheus operator) | `false`
+| `prometheus.metricServer.podMonitor.interval`              | Scraping interval for metric server using podMonitor crd (prometheus operator) | ``
+| `prometheus.metricServer.podMonitor.scrapeTimeout`         | Scraping timeout for metric server using podMonitor crd (prometheus operator) | ``

```

Comment:
```suggestion
| `prometheus.metricServer.podMonitor.scrapeTimeout`         | Scraping timeout for metric server using Pod Monitor CRD (Prometheus operator) | ``

```

Reply:
See first comment. This was resolved after our Slack discussion.